packages/contracts-bedrock/interfaces/L1/IOptimismPortalInterop.sol

```diff
@@ -2,49 +2,48 @@
 pragma solidity ^0.8.0;
 
 import { Types } from "src/libraries/Types.sol";
-import { GameType, Timestamp } from "src/dispute/lib/LibUDT.sol";
+import { GameType } from "src/dispute/lib/LibUDT.sol";
 import { IDisputeGame } from "interfaces/dispute/IDisputeGame.sol";
 import { IDisputeGameFactory } from "interfaces/dispute/IDisputeGameFactory.sol";
 import { ISystemConfig } from "interfaces/L1/ISystemConfig.sol";
 import { ISuperchainConfig } from "interfaces/L1/ISuperchainConfig.sol";
 import { ConfigType } from "interfaces/L2/IL1BlockInterop.sol";
+import { IAnchorStateRegistry } from "interfaces/dispute/IAnchorStateRegistry.sol";
 
 interface IOptimismPortalInterop {
-    error AlreadyFinalized();
-    error BadTarget();
-    error Blacklisted();
-    error CallPaused();
     error ContentLengthMismatch();
     error EmptyItem();
-    error GasEstimation();
     error InvalidDataRemainder();
-    error InvalidDisputeGame();
-    error InvalidGameType();
     error InvalidHeader();
-    error InvalidMerkleProof();
-    error InvalidProof();
-    error LargeCalldata();
-    error NonReentrant();
+    error OptimismPortal_AlreadyFinalized();
+    error OptimismPortal_BadTarget();
+    error OptimismPortal_CallPaused();
+    error OptimismPortal_CalldataTooLarge();
+    error OptimismPortal_GasEstimation();
+    error OptimismPortal_GasLimitTooLow();
+    error OptimismPortal_ImproperDisputeGame();
+    error OptimismPortal_InvalidDisputeGame();
+    error OptimismPortal_InvalidMerkleProof();
+    error OptimismPortal_InvalidOutputRootProof();
```

Comment:
if we did onchain error checking, would we catch the bug by changing the name of the error?

Reply:
Hmmm what do you mean? I pretty confident that we don't do any onchain error checks but I'm not sure about third parties.